CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ We're glad you want to contribute to a Engine Yard v7 Cookbooks repository ! Thi
 ## Instructions for creating a cookbook
 
 1. Clone the repository or pull the latest changes! `git clone` or `git fetch`.
-2. Install cookstyle this can be done with `gem install cookstyle`. This is required for the linting process.
+2. Install cookstyle this can be done with `gem install cookstyle`. This is required for the linting process. (Cook style version **1.25.1 or higher**
```

Comment:
Let's have this text be something like "Make sure cookstyle is version 1.25 or higher".